src/components/TradeWidget/Price.tsx

```diff
@@ -21,17 +25,26 @@ const Wrapper = styled.div`
 
   > strong {
     display: flex;
+    align-items: center;
     text-transform: capitalize;
     color: var(--color-text-primary);
     width: 100%;
     margin: 0 0 1rem;
     padding: 0;
     box-sizing: border-box;
     font-size: 1.5rem;
+
     @media ${MEDIA.mobile} {
       font-size: 1.3rem;
     }
 
+    > ${FormMessage} {
+      width: min-content;
+      white-space: nowrap;
+      font-size: x-small;
```

Comment:
Do we ever use font-size keywords? We usually do `*(r)em`, no? Not that it matters